src/lib/isPostalCode.js

```diff
@@ -12,7 +12,7 @@ const patterns = {
   AU: fourDigit,
   BE: fourDigit,
   BG: fourDigit,
-  BR: /^\d{5}-\d{3}$/,
+  BR: /^\d{5}(-\d{3})$/,
```

Comment:
@bloodf added a regex where it made sure that it will allow -, . , | as a separator.
From my own research, I realized there are no cases where the separator was not -.
That is why I removed some of his changes.

Cc. @profnandaa.

Reply:
What's the use of the brackets  `(, )`?

Reply:
To catch the 
```javascript 
-\d{3}
``` 
as a group.

Reply:
But, I don't know what it means by `,` you have made in the comment.

Reply:
Was wondering why we need to catch the group, I'm I missing something? 🤔

Reply:
That is true @profnandaa, I have updated it.